### ISSUE TITLE
Question for cellcomm？

### ISSUE THREAD

**Issue by Farewellznm** [edroaldo/fusca#18]
Hello，May I ask a question about cellcomm?

when I use the fuction `findpaths.simpleRJava` for Find paths in PBMC.rds, it gives me a reminder lack of file 'FlowNetwork_all_paths_subnet.txt'.I loaded the PPI network (`lr_network`) correctly in the process, not quite sure if it was due to the path or something else？

```
 cellcomm <- findpaths.simpleRJava(cellcomm, sources.targets = sts2$receptor[[interaction]], file = filename, dir = dirname, maindir = maindir)
[1] "CD8_CD14.Monocytes.txt"
[1] "H:/CellComm"
[1] "NET=/H:/CellComm/CD8_CD14.Monocytes.txt"
Creating flow network...
Error in file(file, "rt") : cannot open the connection
In addition: Warning message:
In file(file, "rt") :
  cannot open file 'FlowNetwork_all_paths_subnet.txt': No such file or directory
```

**Comment by edroaldo** [comment]
Hello,

Thanks for trying FUSCA!

Can you please indicate which tutorial you are using? The PBMC tutorial
here
https://github.com/edroaldo/fusca/blob/main/tutorial/FUSCA_tutorial_3k_PBMC_ds.ipynb
is not supposed to use lr_network. Now, if you are using the cellcomm
tutorial here
https://github.com/edroaldo/fusca/blob/main/tutorial/CellComm_ST_tutorial.ipynb,
I need more details about your script.

Can you please provide me additional code after step 18 from the last
tutorial "Predict a gene regulatory network (GRN) and create/define a
protein-protein interaction network (PPI) for intracellular signaling." up
to the error you are getting.

Also, please check the potential solution here. Maybe this already helps.
https://github.com/edroaldo/fusca/issues/16

Thank you! Please, let me know if you have any questions.

**Comment by Farewellznm** [comment]
Thank you for your reply！And I checked the potential solution，but I'm still embarrassed to ask you for help.

I use the tutorial of **CellComm for ST data** as a reference for scRNAseq data ( PBMC).'  As in your article, I would like to use single cell transcriptome data to predict downstream pathways using cellcomm.

So I changed the input parameters of the data. Such as, `CreateCellRouter(... , assay.type = "ST"...)`
 to  `CreateCellRouter(... , assay.type = "RNA"...)`. 

I've uploaded the code I tried as an attachment and marked where the error was at the end！
[cellcomm_help.txt](https://github.com/edroaldo/fusca/files/9653448/cellcomm_help.txt)

Thank you! Hope to get your reply！
